en/User interface/Use tabs in Obsidian.md

```diff
@@ -0,0 +1,108 @@
+---
+alias: 
+- Stacked tabs
+- Linked pane
+- Pane layout
+---
+
+You can open as many tabs as you want in Obsidian. You can also arrange tabs to create custom layouts that persist until the next time you open the application.
+
+## Open a new tab
+
+At the top of the application window, next to the last tab on the right, select **New tab** (plus icon). Or, use a keyboard shortcut:
+
+- **Windows and Linux:** `Ctrl+t`
+- **macOS:** `Cmd+t`
+
+## Open a new tab in a new tab group
+
+To open a tab in a new tab group, use the following keyboard shortcuts:
+
+- **Windows and Linux:** `Ctrl+Shift+t`
+- **macOS:** `Cmd+Shift+t`
+
+## Open a link
+
+Select a link in Obsidian to open it in the active tab.
+
+To open a link in a new tab, press `Ctrl` (or `Cmd` on macOS) and select the link.
+
+## Organize your tabs and windows
+
+Every tab belongs to a _tab group_, and every tab group belongs to a window. You can move tabs into or out of tab groups and from the application window into their own windows.
```

Comment:
Something about this section feels a bit off. I don't think this part is necessary:

> and every tab group belongs to a window

I also wonder if it makes sense to introduce the concept of a tab group but not really say what it is.

Here's my attempt:

```suggestion
Every tab belongs to a _tab group_. You can drag and drop tabs to rearrange them within their tab group, move them to a different tab group, or create a new tab group. On desktop, tabs can be dragged out of their window to open them in their own _popout window_.
```